<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homeFrame</class>
 <widget class="QDialog" name="homeFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>plantpal</string>
  </property>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#background{
background-color: #a2b5b0;}</string>
   </property>
   <widget class="QWidget" name="waterWgt" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>500</width>
      <height>190</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
background-color: #C5DCD6;</string>
    </property>
    <widget class="QLabel" name="waterLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>500</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato Semibold</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 10px;
background-color: #d6efe8;
border-top-left-radius: 15px;
border-top-right-radius: 15px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;
</string>
     </property>
     <property name="text">
      <string>Needs Watering</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>49</y>
       <width>501</width>
       <height>131</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 5px;</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>501</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="addPlantBtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
font: &quot;Lato&quot;;
border: 4px solid '#bbd1cb';
border-radius: 10px;
font-size: 20px;
background-color: #bbd1cb
}
*:hover{
border: 4px solid '#b4c8c2';
background-color:'#b4c8c2';
}
</string>
    </property>
    <property name="text">
     <string>Add Plant</string>
    </property>
   </widget>
   <widget class="QWidget" name="plantsWgt" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>329</y>
      <width>500</width>
      <height>410</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #C5DCD6;
border-radius: 15px;</string>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>16</y>
       <width>501</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
background-color: #C5DCD6;</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>501</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
